lab-cameron/lib/greeting.js

```diff
@@ -0,0 +1,11 @@
+'use strict';
+
+module.exports = exports = {};
+
+// Stretch Goal: typing 'node greeting.js < your name >'
```

Comment:
Comments are great! Get in the habit of establishing a workflow where you have a branch that contains your comments and the branch you submit to me doesn't have comments. :)